bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/bno/NotifyMemberFlow.kt

```diff
@@ -18,12 +18,13 @@ data class OnMembershipChanged(val changedMembership : StateAndRef<Membership.St
 @CordaSerializable
 data class OnMembershipRevoked(val revokedMember : Party)
 
-class NotifyActiveMembersFlow(private val notification : Any) : FlowLogic<Unit>() {
+class NotifyActiveMembersFlow(private val notification : Any) : BusinessNetworkAwareInitiatingFlow<Unit>() {
+
     @Suspendable
-    override fun call() {
-        val databaseService = serviceHub.cordaService(DatabaseService::class.java)
-        databaseService.getActiveMemberships().forEach { subFlow(NotifyMemberFlow(notification, it.state.data.member)) }
+    override fun callWithActiveMembershipList(memberships: List<StateAndRef<Membership.State>>) {
```

Comment:
Should we simplify to `callWithMembershipsList()`?

Reply:
well, that's what I had first but I thought to make it very clear this is active memberships why not stick it in the name

Reply:
Or maybe even callWithMembers() or something. To make it super short and simple